app/controllers/nonsipp/employercontributions/EmployerContributionsMemberListController.scala

```diff
@@ -227,13 +226,18 @@ object EmployerContributionsMemberListController {
     FormPageViewModel(
       title = Message(title, memberList.size),
       heading = Message(heading, memberList.size),
-      description = Some(ParagraphMessage("employerContributions.MemberList.paragraph")),
+      description = None,
       page = ActionTableViewModel(
-        inset = "employerContributions.MemberList.inset",
+        inset = ParagraphMessage(
+          "employerContributions.MemberList.paragraph1"
+        ) ++
+          ParagraphMessage(
+            "employerContributions.MemberList.paragraph2"
+          ),
         head = Some(List(TableElem("Member name"), TableElem("Status"))),
         rows = rows(srn, mode, memberList, userAnswers),
         radioText = Message("employerContributions.MemberList.radios"),
-        showRadios = memberList.length < maxNotRelevant,
+        showInsetWithRadios = true,
```

Comment:
is it always true and no limit to add a new one ?

Reply:
Yes, here it is always true, as the question is "Have you finished....?"